utils/src/deployer.rs

```diff
@@ -190,7 +190,7 @@ pub async fn deploy_light_client_contract<M: Middleware + 'static>(
     let plonk_verifier = contracts
         .deploy_tx(
             Contract::PlonkVerifier,
-            PlonkVerifier2::deploy(l1.clone(), ())?,
+            PlonkVerifier::deploy(l1.clone(), ())?,
```

Comment:
@alxiong @alysiahuggins I think this now needs to be done via a smart contract upgrade (resp. downgrade). Changing it here in the code is not enough to use it on decaf and mainnet.

Reply:
update: Since we are redeploying it, we would just run this.
(i did a project wise search, I don't see any accidental use of PV2 anymore)